### ISSUE TITLE
Keyboard not working when interacting though OBS

### ISSUE THREAD

**Issue by x10power** [jeanwll/LiveSplitOne-OBS-Layout#1]
Hello,

I been toying around with this today and I am still trying to figure out how to get my keyboard to function the LiveSplit One. If I click in any of the text fields to enter the game it works but for some reason when I try to use the shortcut keys they aren't working at all, to use or rebind. 

Thanks,

**Comment by jeanwll** [comment]
LiveSplit One hotkeys are not working in background.
I am currently writing a small program to allow using/binding global hotkeys.
It should be done very soon, I will update this repo when it's ready.

**Comment by jeanwll** [comment]
I just released https://github.com/jeanwll/LiveSplitOne-GlobalHotkeys
Tell me if it works for you!

**Comment by x10power** [comment]
Yeah once I figure out how and where to put everything I will let you know. Thanks for the quick reply!

**Comment by x10power** [comment]
Yeah for some reason I can't get it to work with the keyboard shortcuts but I came up with a way I can use it in the mean time while you are working on it so I can at least reset the timer when needed!

**Comment by jeanwll** [comment]
I would like to know more about the issues you're facing, maybe you can tell me more on discord? Twix#3005

**Comment by x10power** [comment]
Do you have the correct discord info with the Twix#3005?

**Comment by x10power** [comment]
I can be reach at x10power#6965
